files/build_templates/sonic_debian_extension.j2

```diff
@@ -64,6 +64,9 @@ clean_sys() {
 trap_push clean_sys
 sudo LANG=C chroot $FILESYSTEM_ROOT mount sysfs /sys -t sysfs
 
+[ -d "/sys/fs/cgroup" ] && [ "$(stat -fc '%T' /sys/fs/cgroup)" = "cgroup2fs" ] && \
+    sudo LANG=C chroot $FILESYSTEM_ROOT mount cgroup2 /sys/fs/cgroup -t cgroup2
```

Comment:
it would be better if you can modify the files/docker/docker incorporate cgroup2 support there.

Reply:
docker.sh is running in chroot, so I can't check fs on /sys/fs/cgroup then.
I think I need additional time to investigate this.

Reply:
@lguohan 
I think new version also will work, but I'm not sure about builds with enabled CROSS_BUILD_ENVIRON. I haven't test them.